src/worker.rs

```diff
@@ -222,20 +230,21 @@ pub async fn worker_main(goose_attack: &GooseAttack) -> GooseAttack {
         .expect("failed to launch GooseAttack")
 }
 
+// Push request or task statistics to manager.
 pub fn push_stats_to_manager(
     manager: &Socket,
-    requests: &HashMap<String, GooseRequest>,
+    requests: Option<HashMap<String, GooseRequest>>,
+    tasks: Option<GooseTaskStats>,
```

Comment:
While I like the option to make arguments optional, is there really a use-case where you want to send requests and not even an empty GooseTaskStats?

Reply:
The enum will remove this Option<>, as we'd always be sending something. The Option<> only existed because we could send either alone, or both together.

Reply:
This is the new function signature:
```rust
pub fn push_metrics_to_manager(
    manager: &Socket,
    metrics: Vec<GooseMetrics>,
    get_response: bool,
) -> bool {
```